queue_services/entity-filer/tests/unit/filing_processors/test_incorporation_filing.py

```diff
@@ -182,30 +187,30 @@ def test_incorporation_filing_process_no_nr(app, session, legal_type, filing, le
         assert parties[1]["officer"]["partyType"] == "organization"
         assert parties[1]["officer"]["organizationName"] == "Xyz Inc."
 
-    mock_get_next_corp_num.assert_called_with(filing["filing"]["incorporationApplication"]["nameRequest"]["legalType"])
+    # TODO: This assert_called_with thinks it gets called with 'P' as legalType for some reason
+    # mock_get_next_corp_num.assert_called_with(filing["filing"]["incorporationApplication"]["nameRequest"]["legalType"])
```

Comment:
This is because we create person identifiers for LE entries where `entity_type` = "person" now.

So this assertion will pick up the last call made to `mock_get_next_corp_num` which was called with "P".

I was testing locally a bit.  You can probably reference the list that contains all the calls to the mock function.  The first item should be the call to get the corp num for the business.

![image](https://github.com/bcgov/lear/assets/6685223/9f46c314-d790-4e5a-9740-5fe7c4abb3a9)